src/test/java/apprenticeship/BoardTest.java

```diff
@@ -90,9 +90,9 @@ public void canRenderGridWithOnlyCellNumberId() throws Exception {
         String newGrid = tictactoe.getGrid();
 
 
-        String expected =   "  1  |  2  |  3  |" + "\n----- ----- ----- \n" +
-                            "  4  |  5  |  6  |" + "\n----- ----- ----- \n" +
-                            "  7  |  8  |  9  |" + "\n----- ----- ----- \n";
+        String expected =   "  0  |  1  |  2  |" + "\n----- ----- ----- \n" +
+                            "  3  |  4  |  5  |" + "\n----- ----- ----- \n" +
+                            "  6  |  7  |  8  |" + "\n----- ----- ----- \n";
 
```

Comment:
@tamminhdao from a developer perspective that makes sense. Does this change make sense to a non-programmer?